docs/data/sql_functions.yml

```
@@ -643,6 +643,9 @@ collection:
   - sql: ARRAY_REVERSE(haystack)
     table: haystack.arrayReverse()
     description: Returns an array in reverse order. If the array itself is null, the function will return null.
+  - sql: ARRAY_SLICE(array, start_offset, end_offset)
+    table: array.arraySlice(start_offset, end_offset)
+    description: Returns a subarray of the input array between 'start_offset' and 'end_offset' inclusive. The offsets are 1-based however 0 is also treated as the beginning of the array. Positive values are counted from the beginning of the array while negative from the end. If 'start_offset' is after 'end_offset' or both 'start_offset' and 'end_offset' are out of array bounds an empty array will be returned. Returns null if any input is null.
```

```suggestion
    description: Returns a subarray of the input array between 'start_offset' and 'end_offset' inclusive. The offsets are 1-based however 0 is also treated as the beginning of the array. Positive values are counted from the beginning of the array while negative from the end. If 'start_offset' is after 'end_offset' or both are out of array bounds an empty array will be returned. Returns null if any input is null.
```
sorry didn't notice it earlier however it could be shortened a bit more

---

ok.